vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -64,6 +64,8 @@ type internal FSharpCheckerProvider
 
     member this.Checker = checker.Value
 
+type Refreshable<'T> = 'T * (unit -> 'T)
```

Comment:
The idea here is that we have a function to recalc the options when we need to, But we also can deliver the latest copy of the options to other callers